samples/testing-frameworks/appium/client-side/csharp/sample-test/Test123/Tests.cs

```diff
@@ -22,13 +22,12 @@ public void BeforeAll()
 			String BITBAR_APIKEY = "<YOUR BITBAR API KEY>";
 
 			AppiumOptions capabilities = new AppiumOptions();
-			capabilities.AddAdditionalCapability("device", "Android");
-
-			capabilities.AddAdditionalCapability("deviceName", "Android");
-			capabilities.AddAdditionalCapability("platformName", "Android");
+			capabilities.AddAdditionalCapability("appium:deviceName", "Android");
+			capabilities.AddAdditionalCapability("appium:platformName", "Android");
```

Comment:
according to https://appium.io/docs/en/2.0/guides/caps/
platformName should be without prefix